planner/core/src/main/java/eu/seaclouds/platform/planner/core/HttpHelper.java

```diff
@@ -83,22 +84,48 @@ public String getRequest(String restPath, List<NameValuePair> params){
         }
     }
 
+    public String postInBody(String restPath, String bodyContent){
+        HttpPost post = new HttpPost(prepareRequestURL(restPath));
+        try {
+            StringEntity entity = new StringEntity(bodyContent);
+            post.setEntity(entity);
+            HttpResponse response = httpclient.execute(post);
+            String result = EntityUtils.toString(response.getEntity());
+            return result;
+        }catch (ClientProtocolException e) {
+            return null;
```

Comment:
I mean, SeaClouds is mad of a bunch of services. What is the expected behaviour in case of failure of this services. Is there an agreed code-style, error handling and notification policy or something similar?  Should each developer do as he like?